numpy/lib/arrayterator.py

```diff
@@ -66,7 +66,7 @@ class Arrayterator:
     Examples
     --------
     >>> a = np.arange(3 * 4 * 5 * 6).reshape(3, 4, 5, 6)
-    >>> a_itor = np.lib.Arrayterator(a, 2)
+    >>> a_itor = np.lib.arrayterator.Arrayterator(a, 2)
```

Comment:
`np.lib.Arrayterator` is still present given the current state of this PR. Was this change needed to make the doc build happy? If not, may be better to revert it.

Reply:
I did it in advance, but `lib` refactor is not a part of this PR - reverted.